src/DynamoManipulation/DynamoManipulationExtension.cs

```diff
@@ -238,7 +238,7 @@ private static bool InspectInputsForNode(NodeModel node)
 
         public void Shutdown()
         {
-            
+            Dispose();   
         }
```

Comment:
If Dispose is called after Shutdown in `DynamoView.WindowClosed`, what's the need  call it here?

Reply:
@aparajit-pratap - this depends on if we leave the ViewExtension disposal calls in or not - @mmisol was suggesting maybe leaving them out to keep the behavior as it was before this PR - if we do do, I can leave this.

If we keep the disposal calls after shutdown, I can remove this.